yubikey-luks-enroll

```diff
@@ -32,6 +36,7 @@ while getopts ":s:d:hcv" opt; do
 		echo 
 		echo " -d <partition>: set the partition"
 		echo " -s <slot>     : set the slot"
```

Comment:
Can you please change this to "set LUKS slot", so that these two slots don't get confused.